index.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./Styles/styles.css" type="text/css">
+    <script src="https://kit.fontawesome.com/c2d0b29889.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Daniel Yerimah's Capstone Project</title>
+</head>
+<body>
+    <header>
+    <div class="topnav">
+        <ul>
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="container">
+    <div class="navbar">
+        <div class="logo">
+            <a href="index.html"><img src="./images/newlogo.png" alt="African Tech Summit Logo"></a>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="border-about"><a href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </div>
+    </div>
+        <div class="bars" id="hambug">
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        </div>
+
+    <div class="toggle-menu">
+            <div><img src="./images/closetoggle.png" alt="close toggle" id="togclose" ></div>
+
+            <div class="hambug-items">
+            <ul class="menu">
+                <li class="list-menu"><a href="about.html" class="a-link">About</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Program</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Join</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Sponsor</a></li>
+                <li class="list-menu"><a href="#" class="a-link">News</a></li>
+                <li class="list-menu"><a href="#" class="a-link">CC Campaign</a></li>
+            </ul>
+            </div>
+
+    </div>
+    </header>
+
+    <section class="hero">
+        <div class="container intro">
+        <h1>"Hello! Tech Africa"</h1>
+        <h2>FRESH START <br> AFRICA TECH SUMMIT 2022</h2>
```

Comment:
The pages should look almost identical to the original design. Please make sure to add text effect for the h2 as with the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).